pkg/cmd/flags/capture.go

```diff
@@ -90,6 +96,37 @@ func PrepareCapture(captureSlice []string) (tracee.CaptureConfig, error) {
 					capture.Net.CaptureCommand = true
 				}
 			}
+			capture.Net.CaptureLength = 0 // proforma: capture headers only by default
+		} else if strings.HasPrefix(cap, "pcap-snaplen:") {
+			context := strings.TrimPrefix(cap, "pcap-snaplen:")
+			var amount int64
+			var err error
+			context = strings.ToLower(context) // normalize
+			if context == "max" {
+				amount = (1 << 16) // sets full packet length for capturing
+			} else if context == "headers" {
+				amount = 0 // sets headers only length for capturing (default)
+			} else if strings.HasSuffix(context, "kb") ||
+				strings.HasSuffix(context, "k") {
+				context = strings.TrimSuffix(context, "kb")
+				context = strings.TrimSuffix(context, "k")
+				amount, err = strconv.ParseInt(context, 10, 64)
```

Comment:
As the final amount is treated as an uint, how about using `strconv.ParseUint()`?